src/SparkFun_SCD4x_Arduino_Library.h

```diff
@@ -35,6 +35,15 @@
 #include <Wire.h>
 #endif
 
+//Enable/disable including debug log (to allow saving some space)
```

Comment:
Please add one extra line here:

```
//#define SCD4x_ENABLE_DEBUGLOG 1 // Uncomment this line to manually disable the debug messages
```

Reply:
On second thoughts, please move the new line to close to the start of the file - to make it more obvious.
Thank you!

Reply:
Fixed